modules/core/src/passes/draw-layers-pass.js

```diff
@@ -2,9 +2,12 @@ import LayersPass from './layers-pass';
 
 export default class DrawLayersPass extends LayersPass {
   // PRIVATE
-  getModuleParameters(layer, pixelRatio) {
+  getModuleParameters(layer, pixelRatio, effects, effectProps) {
     const moduleParameters = super.getModuleParameters(layer, pixelRatio);
-    Object.assign(moduleParameters, this.getObjectHighlightParameters(layer));
+    Object.assign(moduleParameters, this.getObjectHighlightParameters(layer), effectProps);
+    for (const effect of effects) {
```

Comment:
* At first glance, it seems a little strange that the DrawLayersPass has knowledge of other effects. I would expect e.g. the renderer to call something like `effectManager.extractGlobalSettingsFromEffects()` before calling the render passes.
* Even if we want to/need to do it here, I would still encapsulate the iteration of the effects inside the effectManager, e.g. `effectManager.getParametersForLayer()`